File: 118-pascals-triangle/118-pascals-triangle.py
class Solution:
    def generate(self, numRows: int) -> List[List[int]]:
        final=[]
        x=[]
        for i in range(numRows):
            num=1
            for j in range(i+1):
                x.append(num)
                num=int(num*(i-j)/(j+1))
                
            final.append(x)
            x=[]
        return final

File: 118-pascals-triangle/118-pascals-triangle.cpp
class Solution {
public:
    vector<vector<int>> generate(int numRows) {
        vector<vector<int>> ans;
        vector<int> temp;
        for(int i=0;i<numRows;i++)
        {
            int num=1;
            for(int j=0;j<=i;j++)
            {
               temp.push_back(num);
                num=(num*(i-j)/(j+1));
            }
            ans.push_back(temp);
            temp.clear();
        }
    return ans;
    }
};